types/rezervo.ts

```diff
@@ -67,6 +67,25 @@ export type ClassPopularityIndex = {
     [recurrentId: string]: ClassPopularity;
 };
 
+export enum IntegrationIdentifier {
+    sit = "sit",
+    fsc = "fsc",
+}
+
+export type RezervoIntegration = {
+    name: string;
+    acronym: IntegrationIdentifier;
+    businessUnits: RezervoBusinessUnit[];
+};
+
+export type RezervoBusinessUnit = {
+    name: string;
+    // eslint-disable-next-line no-unused-vars
+    weekScheduleFetcher: (weekNumber: number) => Promise<any>;
+    // eslint-disable-next-line no-unused-vars
+    weekScheduleAdapter: (weekSchedule: any) => RezervoWeekSchedule;
```

Comment:
Would be really cool to use generic here, for completeness. However, I am not sure how one would do this.

It is not strictly necessary tho, since changes to the WeekSchedule types would trigger TS errors anyways.